File: Backend/maintenace-service/src/Entity/Bodega.cs
namespace Entity
{
    public class Bodega
    {
        public string? Id { get; set; }              // Identificador único de la bodega
        public string? IdPlanta { get; set; }        // FK de la planta asociada
        public string? Nombre { get; set; }          // Nombre de la bodega
        public bool? Estado { get; set; }            // Estado (activo/inactivo)
        public DateTime? Fecha_log { get; set; }     // Fecha de creación o última modificación
    }
}

File: Frontend/src/Types/bodega.ts
export interface Bodega {
  id: string;
  idPlanta: string;
  nombre: string;
  estado: boolean;
  fecha_log: string;
}

export interface BodegaResponse {
  data: Bodega[];
  message: string;
  success: boolean;
}

export interface BodegaCreateRequest {
  idPlanta: string;
  nombre: string;
  estado: boolean;
  fecha_log: string;
}

export interface BodegaUpdateRequest extends BodegaCreateRequest {
  id: string;
}
